go.mod

```diff
@@ -34,6 +34,4 @@ require (
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.2-alpha.regen.1
 
-replace github.com/keybase/go-keychain => github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4
```

Comment:
so now we are going back to the keybase version? can we not just use 99designs by default? sorry i'm a bit confused about this

Reply:
Nope. We'll just remove this replace statement that although it hides a compilation warning, it does not fix any actual problem.